admin_guide/diagnostics_tool.adoc

```diff
@@ -68,14 +71,77 @@ Non-standard locations can be specified with flags (respectively,
 `--config`, `--master-config`, and `--node-config`). If a configuration file
 is not found or specified, related diagnostics are skipped.
 
-Consult the output with the `--help` flag for all available options.
+Available diagnostics include:
+
+[options="header"]
+|===
+
+|Diagnostic Name |Purpose
+
+|AggregatedLogging
+|Check the aggregated logging integration for proper configuration and operation.
+
+|AnalyzeLogs
+|Check systemd service logs for problems. Does not require a configuration file to check against.
+
+|ClusterRegistry
+|Check that the cluster has a working Docker registry for builds and image streams.
+
+|ClusterRoleBindings
+|Check that the default cluster role bindings are present and contain the expected subjects according to base policy.
+
+|ClusterRoles
+|Check that cluster roles are present and contain the expected permissions according to base policy.
+
+|ClusterRouter
+|Check for a working default router in the cluster.
+
+|ConfigContexts
+|Check that each context in the client configuration is complete and has connectivity to its API server.
+
+|DiagnosticPod
+|Creates a pod that runs diagnostics from an application standpoint, which checks that DNS within the pod is working as expected and the credentials for the default service account authenticate correctly to the master API.
+
+|EtcdWriteVolume
+|Check the volume of writes against etcd for a time period and classify them by operation and key. This diagnostic only runs if specifically requested, because it does not run as quickly as other diagnostics and can increase load on etcd.
+
+|MasterConfigCheck
+|Check this host's master configuration file for problems.
+
+|MasterNode
+|Check that the master running on this host is also running a node to verify that it is a member of the cluster SDN.
+
+|MetricsApiProxy
+|Check that the integrated Heapster metrics can be reached via the cluster API proxy.
+
+|NetworkCheck
+|Create diagnostic pods on multiple nodes to diagnose common network issues from an application standpoint. For example, this checks that pods can connect to services, other pods, and the external network.
+
+If there are any errors, this diagnostic stores results and retrieved files in a local directory (*_/tmp/openshift/_*, by default) for further analysis. The directory can be specified with the `--network-logdir` flag.
```

Comment:
NetworkCheck has a longer description than the rest. I assume that these additional details render okay in a local build.

Reply:
Does indeed!